src/main/java/com/microsoft/sqlserver/jdbc/SQLServerLob.java

```diff
@@ -21,4 +21,13 @@
      * @throws SQLException
      */
     abstract void fillFromStream() throws SQLException;
+
+    protected boolean delayLoadingLob = true;
+    /**
+     * Provides functionality for the result set to set whether to load the LOB objects fully. Setting this property to
+     * TRUE will cause LOBs to be loaded into memory. The default behavior is FALSE.
+     */
+    void setDelayLoadingLob() {
```

Comment:
Shouldn't this setter take in a boolean value, so that the user can go back and forth between setting this to true or false?  The comment also seems to suggest this setter should be able to provide that functionality.

Reply:
Also since other `setDelayLoadingLobs`s have an `s` at the end, add an `s` to this method's name as well.

Reply:
Not necessary, this controls the option for the single LOB. The ResultSet will read the property from the Connection object and determine whether to call this API or not. The user is able to toggle it on and off with the current logic.